src/main/java/choaticbot/ChoaticBot.java

```diff
@@ -0,0 +1,128 @@
+package choaticbot;
+
+import java.util.Scanner;
+import java.io.*;
+import java.nio.file.*;
+
+import choaticbot.actions.ActionManager;
+import choaticbot.actions.Action;
+import choaticbot.exceptions.ChoaticBotException;
+import choaticbot.inputs.InputProcessor;
+import choaticbot.inputs.ProcessedInput;
+import choaticbot.tasks.TaskList;
+import choaticbot.tasks.Task;
+import choaticbot.tasks.ToDos;
+import choaticbot.tasks.Deadlines;
+import choaticbot.tasks.Events;
+
+import static choaticbot.ui.Ui.printText;
+
+public class ChoaticBot {
+    public TaskList tasklist;
+    private static final String DATA_FILE_PATH = "./data/choaticbot.txt";
+
+    public ChoaticBot() {
+        this.tasklist = new TaskList();
+        loadTasks();
+    }
+
+    public static void welcome() {
+        String welcomeMsg = "Hello! I'm ChoaticBot\n" + "What can I do for you?\n";
+        printText(welcomeMsg);
+    }
+
+    public static void bye() {
+        String byeMsg = "Bye. Hope to see you again soon!\n";
+        printText(byeMsg);
+    }
+
+    private void loadTasks() {
+        try {
+            Files.createDirectories(Paths.get("./data"));
+            File file = new File(DATA_FILE_PATH);
+            if (file.exists()) {
+                BufferedReader reader = new BufferedReader(new FileReader(file));
+                String line;
```

Comment:
Perhaps a more specific name to this String would be clearer to the reader as to what line we are looking at?